dsa/tests/privatekey.rs

```diff
@@ -25,7 +25,7 @@ fn decode_encode_openssl_private_key() {
     assert!(private_key.is_valid());
 
     let reencoded_private_key = private_key
-        .to_pkcs8_pem(LineEnding::LF)
+        .to_pkcs8_pem(LineEnding::default())
```

Comment:
I'd suggest keeping these on `LineEnding::LF` so the tests pass consistently regardless of operating system

Reply:
There was this CI failure [here](https://github.com/RustCrypto/signatures/runs/6434988350?check_suite_focus=true) which failed because the OpenSSL PEM file had CRLF line endings (the checkout action probably changed the line endings). That's why I adjusted it to using `LineEnding::default()` 

But yes, I should probably adjust the CI instead of changing the tests